diff --git a/node_modules/react-native-web/dist/vendor/react-native/Animated/Animated.js b/node_modules/react-native-web/dist/vendor/react-native/Animated/Animated.js
index b4ead3e..c1b4e92 100644
--- a/node_modules/react-native-web/dist/vendor/react-native/Animated/Animated.js
+++ b/node_modules/react-native-web/dist/vendor/react-native/Animated/Animated.js
@@ -16,14 +16,15 @@ import ScrollView from './components/AnimatedScrollView';
 import SectionList from './components/AnimatedSectionList';
 import Text from './components/AnimatedText';
 import View from './components/AnimatedView';
-import * as AnimatedMock from './AnimatedMock';
-import * as AnimatedImplementation from './AnimatedImplementation';
+import AnimatedMock from './AnimatedMock';
+import AnimatedImplementation from './AnimatedImplementation';
 var Animated = Platform.isTesting ? AnimatedMock : AnimatedImplementation;
-export default _objectSpread({
+export default {
   FlatList,
   Image,
   ScrollView,
   SectionList,
   Text,
-  View
-}, Animated);
\ No newline at end of file
+  View,
+  ...Animated,
+};
\ No newline at end of file
