diff --git a/node_modules/react-native-web/dist/cjs/vendor/react-native/Animated/Animated.js b/node_modules/react-native-web/dist/cjs/vendor/react-native/Animated/Animated.js
index 191513a..16547b6 100644
--- a/node_modules/react-native-web/dist/cjs/vendor/react-native/Animated/Animated.js
+++ b/node_modules/react-native-web/dist/cjs/vendor/react-native/Animated/Animated.js
@@ -23,9 +23,9 @@ var _AnimatedText = _interopRequireDefault(require("./components/AnimatedText"))
 
 var _AnimatedView = _interopRequireDefault(require("./components/AnimatedView"));
 
-var AnimatedMock = _interopRequireWildcard(require("./AnimatedMock"));
+var AnimatedMock = _interopRequireDefault(require("./AnimatedMock"));
 
-var AnimatedImplementation = _interopRequireWildcard(require("./AnimatedImplementation"));
+var AnimatedImplementation = _interopRequireDefault(require("./AnimatedImplementation"));
 
 /**
  * Copyright (c) Meta Platforms, Inc. and affiliates.
diff --git a/node_modules/react-native-web/dist/vendor/react-native/Animated/Animated.js b/node_modules/react-native-web/dist/vendor/react-native/Animated/Animated.js
index b4ead3e..554cadb 100644
--- a/node_modules/react-native-web/dist/vendor/react-native/Animated/Animated.js
+++ b/node_modules/react-native-web/dist/vendor/react-native/Animated/Animated.js
@@ -16,8 +16,8 @@ import ScrollView from './components/AnimatedScrollView';
 import SectionList from './components/AnimatedSectionList';
 import Text from './components/AnimatedText';
 import View from './components/AnimatedView';
-import * as AnimatedMock from './AnimatedMock';
-import * as AnimatedImplementation from './AnimatedImplementation';
+import AnimatedMock from './AnimatedMock';
+import AnimatedImplementation from './AnimatedImplementation';
 var Animated = Platform.isTesting ? AnimatedMock : AnimatedImplementation;
 export default _objectSpread({
   FlatList,
